### PR TITLE
[Enhancement] finishing touches

### PR DESCRIPTION
**Pull request by Mxrcon**
Hey @abhi18av do you think that after the #2  got merged, and with this content, this pipeline is ready for a README with instructions? I'll add them to this PR and conclude my finishing touches. I've checked every module and found some errors. also, if I'll proceed adding test instructions to each module.

## PR content
* update to gitignore
* tweaks on trimommatic and tb_profiler_collate
* dag.png
* updates to parameters files
* fixes on tbprofiler modules

## PR ending
Hope we have a merge! Feel free to discuss this changes with me, I'm at your disposal to make any necessary change.

### CODE REVIEW

File: modules/tb_profiler/tb_profiler_collate.nf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 //based on https://github.com/nf-modules/tb-profiler/blob/master/main.nf
 nextflow.enable.dsl = 2
 
-params.results_dir = "${params.outdir}/tbprofiler/collate"
+params.results_dir = "${params.outdir}/tb_profiler/collate"
```
Comment (abhi18av) [inline modules/tb_profiler/tb_profiler_collate.nf]: This needs to be consistent with the module name and the file name.

The module name is `TBPROFILER_COLLATE` 
The filename should be `/modules/tbprofiler/collate.nf`
The `results_dir` name should be `tbprofiler/collate`.

File: modules/trimmomatic/trimmomatic.nf [hunk 2]
```diff
@@ -22,7 +22,7 @@ process TRIMMOMATIC {
     fq_2_paired = genomeName + '_R2.p.fastq.gz'
     fq_2_unpaired = genomeName + '_R2.s.fastq.gz'
 
-    def adapterFile = "/usr/local/share/trimmomatic-0.35-6/adapters/NexteraPE-PE.fa"
+    def adapterFile = "/usr/local/share/trimmomatic-0.35-6/adapters/${params.adapter_file_name}"
```
Comment (abhi18av) [inline modules/trimmomatic/trimmomatic.nf:26]: This needs to be added to the `params.yml` file, also I think that perhaps we need to add a default value within the module as well.

For example

```
params.adapter_file_name = "NexteraPE-PE.fa"
```

**Review by abhi18av — CHANGES_REQUESTED** [review]
Thanks for the follow-up on this @Mxrcon  👏 

I have added some comments, mostly related to the patterns that we are following in this repo.

File: modules/tb_profiler/tb_profiler_collate.nf (hunk 1 above)
Comment (Mxrcon) [reply modules/tb_profiler/tb_profiler_collate.nf]: ok! I'll fixup all those names

**Comment by Mxrcon** [comment]
@abhi18av I think that my last commit updates the last requested changes, so I'm gently requesting  new review for this PR

**Review by abhi18av — APPROVED** [review]
Looks good to merge, thanks for accomodating the changes!

**Pull request merged**